Indicators/IndicatorStatus.cs

```diff
@@ -34,5 +34,10 @@ public enum IndicatorStatus
         /// The indicator encountered a math error during calculations
         /// </summary>
         MathError,
+
+        /// <summary>
+        /// The indicator encountered a composite value is not ready
```

Comment:
typo? `The indicator value is not ready`

Reply:
done